podvm-mkosi/build-s390x-image.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -eu
+
+pushd build
+
+workdir=.
+tmp_img_path="$workdir/tmp.qcow2"
+tmp_nbd=/dev/nbd1
+dst_mnt=./dst_mnt
+disksize=100G
+
+qemu-img create -f qcow2 "$tmp_img_path" $disksize
```

Comment:
I would recommend to run shellcheck on the script to get consistent quoting

Reply:
Sure. I run `shellcheck -a -o all build-s390x-image.sh` to check and improve the script.